vector/v.lrs/v.lrs.create/main.c

```diff
@@ -206,14 +206,14 @@ int main(int argc, char **argv)
     driver_opt->required = NO;
     driver_opt->description = _("Driver name for reference system table");
     driver_opt->options = db_list_drivers();
-    driver_opt->answer = db_get_default_driver_name();
+    driver_opt->answer = (char *) db_get_default_driver_name();
```

Comment:
Maybe `option->answer` in [include/gis.h](https://github.com/OSGeo/grass/blob/master/include/gis.h#L541) can be changed to `const char *`? Together with `option->answers` in  [include/gis.h](https://github.com/OSGeo/grass/blob/master/include/gis.h#L543). Something for GRASS 8.

IMHO, these type casts avoid compiler warnings but do not fix the real problem.

Reply:
I don't know the reason Option->answer is not a const string. But I'm afraid changing that wouldn't be a stroll in the park :)  These casts are all over. A alternative now, for this `db_get_default_driver_name()` case, could be to use G_store() (with the memory leak that comes with it).

Reply:
Just out of curiosity I tried compile with Option->answer defined as `const char *answer`.
This resulted in some 660 warnings of type `-Wincompatible-pointer-types-discards-qualifiers`.

Reply:
OK for your change, but still, type casts avoid compiler warnings but do not fix the real problem.